ui/README.md

```diff
@@ -3,58 +3,52 @@ Dashboard made with [elm-bootstrap](http://elm-bootstrap.info/).
 
 ## Install
 
-### Install GUI as a part of Mainflux build
+### Docker container GUI build
 
-Install Elm (https://guide.elm-lang.org/install.html) and then
-
-```
-git clone https://github.com/mainflux/mainflux
-cd mainflux/ui
-make
-```
-
-This will produce `index.html` in the _ui_ directory. In order to use it, `cd`
-to _ui_ and do
-
-`make run`
+Install Docker (https://docs.docker.com/install/) and Docker compose
+(https://docs.docker.com/compose/install/), `cd` to Mainflux root directory and
+then
 
-### Build a standalone native GUI
+`docker-compose -f docker/docker-compose.yml up`
 
-Install Elm (https://guide.elm-lang.org/install.html), `cd` to _ui_ and then
+if you want to launch a whole Mainflux docker composition, or just
 
-`elm make --optimize src/Main.elm`
+`docker-compose -f docker/docker-compose.yml up ui`
 
-This will produce `index.html` in the _ui_ directory. In order to use it do
+if you want to launch just GUI.
 
-`make run`
+### Native GUI build
 
-### About Elm `make`
+Install Elm (https://guide.elm-lang.org/install.html) and then run the following
+commands:
 
-`make` does `elm make src/Main.elm`.
+```
+git clone https://github.com/mainflux/mainflux
+cd mainflux/ui
+make
+```
 
-`make run` just executes `elm reactor`. You can execute `elm reactor` in other
-terminal window and keep it running, and then see changes as you change-compile
-in the first window. You can even use something as
-[entr](http://eradman.com/entrproject/) to have your source compiled
-automatically when you change and save some files.
+This will produce `index.html` in the _ui_ directory. Still in the _mainflux/ui_
+folder, enter
 
-### Build as a part of Docker composition
+`make run`
 
-Install Docker (https://docs.docker.com/install/) and Docker compose
-(https://docs.docker.com/compose/install/), `cd` to Mainflux root directory and
-then
+and follow the instructions on screen.
 
-`docker-compose -f docker/docker-compose.yml up`
+**NB:** `make` does `elm make src/Main.elm` and `make run` executes `elm
+reactor`. Cf. _Makefile_ for more options.
```

Comment:
Meaning of Cf. abbreviation is not so common knowlegde

Reply:
Let the developers learn a bit of humanities language :)